zmon_worker_monitor/builtins/plugins/http.py

```diff
@@ -41,7 +35,19 @@ def configure(self, conf):
         Called after plugin is loaded to pass the [configuration] section in their plugin info file
         :param conf: configuration dictionary
         """
-        return
+        # will use OAUTH2_ACCESS_TOKEN_URL environment variable by default
+        # will try to read application credentials from CREDENTIALS_DIR
+        tokens.configure()
+        tokens.manage('uid', ['uid'])
+
+        token_configuration = conf.get('oauth2.tokens')
+
+        if token_configuration:
+            for part in token_configuration.split(':'):
+                token_name, scopes = tuple(part.split('=', 1))
```

Comment:
I suggest protecting (skipping) token_name ``uid``, to avoid overriding the default scope from the config.